velox/functions/sparksql/String.h

```diff
@@ -845,4 +845,103 @@ struct TranslateFunction {
   }
 };
 
+/// Decodes `string` in 'application/x-www-form-urlencoded' format using UTF-8
+/// encoding scheme.
+template <typename T>
+struct UrlDecoderFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  // ASCII input does NOT always produce ASCII result.
+  static constexpr bool is_default_ascii_behavior = false;
+
+  int decode(StringView input) {
+    assert(input.size() == 2);
+    auto toIntFromHex = [](char x) {
+      switch (x) {
+        case '0':
+          return 0;
```

Comment:
if (std::isdigit(x)) {
return x - '0';}
similar to alphabet char

Reply:
Yes, it can make the code concise. Just fixed. Thanks!